doctor/tasks.py

```diff
@@ -22,11 +22,16 @@
 from seal_rookery.search import seal, ImageSizes
 
 from doctor.lib.mojibake import fix_mojibake
+from doctor.lib.ocr_utils import (
```

Comment:
I think `ocr_utils` might not be quite right. Maybe text_extraction.py or something along those lines?